sourcemash/frontend/static/logotext.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can we crop these images down somehow? It will make it easier to incorporate them into the css.